packaging/Makefile

```diff
@@ -0,0 +1,18 @@
+NAME    := libfabric
```

Comment:
Ditto.  FWIW, from a packaging repo, you can run `$path/to/packaging-repo-clone/install` to copy only the necessary files to the packaging repo.

Reply:
Ok, this removed the following files from packaging folder:
Jenkinsfile
Makefile
README.md
install
packaging